packages/server/app.ts

```diff
@@ -131,6 +131,9 @@ function buildApolloSubscriptionServer(
  * Define mocking config in dev env
  * https://www.apollographql.com/docs/apollo-server/v3/testing/mocking
  */
+
+const roles = ['owner', 'contributor', 'reviewer']
```

Comment:
We already have all roles as constants in `mainConstants.ts` and also `@speckle/shared`